tools/rna_tools/remurna/remurna.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="remurna2" name="remuRNA" version="1.0.0">
+    <description> Measurement of Single Nucleotide Polymorphism induced Changes of RNA Conformation </description>
+    <requirements>
+                <requirement type="package" version="1.2">rnasnp</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+    temp_input=\$(mktemp) && 
+    python $__tool_directory__/create_input_file.py '$seq' '$snp' '\$temp_input' &&
+    \$(which remuRNA)
```

Comment:
Why is this needed? remuRNA should be on your path with Conda already. So just write `remuRNA`